### DISCUSSION TITLE
Warning: you should not extend "group:" presets

### DISCUSSION THREAD

**Discussion by himanshumehta1114** [renovatebot/renovate#9868] [Request Help] [answered]
Hello folks,

I am currently using the following configuration:
```
 {
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["config:base",  "group:allNonMajor", ":automergePatch"],
  "rangeStrategy": "bump",
  "dependencyDashboard": true,
  "prHourlyLimit": 10,
  "gitLabAutomerge": true
}
```

I want to group all my non-major dependencies but auto-merge only `patch` updates. Since yesterday, I starting getting a warning message: `you should not extend "group:" presets`.

Does this mean I will need to update the `packageRules` with the configurations from `group:allNonMajor` preset manually instead of directly mentioning the preset name in extends array?

**Comment by rarkins** [comment, marked as answer]
That seems like an erroneous warning. I'll try to reproduce and fix.

**Reply by HonkingGoose** [reply]
It seems this was fixed with the merge of PR #9871?

**Reply by viceice** [reply]
Yes, should be

**Reply by himanshumehta1114** [reply]
Yeah, not getting warnings now. Thanks for fixing 👍
